packages/blog-starter-kit/themes/enterprise/next.config.js

```diff
@@ -1,5 +1,5 @@
 const ANALYTICS_BASE_URL = 'https://hn-ping2.hashnode.com';
-const ADVANCED_ANALYTICS_BASE_URL = 'https://stats.hashnode.com';
+const ADVANCED_ANALYTICS_BASE_URL = 'https://user-analytics.hashnode.com';
```

Comment:
This is currently used for proxying `/api/collect` (https://github.com/mindsdb/hashnode-starter-kit/pull/17/files#diff-a9b0cf02522cd2a8797011d00973b99b76dc5ce6b23570c6731a3d38d3d7b211R40-R41) but the new analytics calls are made to `/api/analytics`.

This
```javascript
{
  source: '/api/collect',
  destination: `${ADVANCED_ANALYTICS_BASE_URL}/api/collect`,
}
```
should be changed to 
```javascript
{
  source: '/api/analytics,
  destination: `${ADVANCED_ANALYTICS_BASE_URL}/api/analytics`,
}
```

Reply:
Thank you @JannikWempe 
I made this update. Please let me know if it is fine now.

Reply:
This part is fixed now. I have cloned the repo to check if everything is working now but I have found several more issues:

- The new `useEffect` is called conditionally due to the `isProd` check ([see here](https://github.com/mindsdb/hashnode-starter-kit/pull/17/files#diff-049e1536cc838d860847f370d4507d0a86e4cb72ba91887741f73d3d4ffcc3ffR21)). This is violating the rules of React hooks.
- You now have 2 `useEffect`s and 3 of 4 calls are made in both of them.

This is the [PR in our starter kit](https://github.com/Hashnode/starter-kit/commit/f543e14caf8cf017416638d8d60a3e1e3a0c1de1#diff-049e1536cc838d860847f370d4507d0a86e4cb72ba91887741f73d3d4ffcc3ff) that introduced the new analytics. You can take it as a reference.